### PR TITLE
CAL1-9 Facility-profile-Alternative phone number

### PR DESCRIPTION
**Pull request by sunilb0575**
#### Which User story does this relate to?
-https://osi-cwds.atlassian.net/browse/CAL1-9

#### Brief feature description
-Check for alternative phone number in facility profile and facility search. 

#### Does this PR block/effect other stories?
-No

#### Do any commands needs to be run in order to run the project with this PR?
-No

#### Code coverage percent diff as a result of this PR?
-No

### CODE REVIEW

File: app/javascript/search/common/commonUtils.js [hunk 1]
```diff
@@ -16,16 +17,26 @@ export const stringForCityStateZip = (address) => {
   return address.address.city + ',' + ' ' + address.address.state + ' ' + address.address.zip_code
 }
 
-export const phoneNumberOrNa = (phone) => {
-  if (phone.length > 0) {
-    return formatPhoneNumberForDashes(phone[0])
+export const getPrimaryNumberOrNA = (phones) => {
```
Comment (lauherkcambria) [inline app/javascript/search/common/commonUtils.js]: these methods look like they be combined into one  by taking in the relation parameter (i.e. 'primary', 'residential', or 'Alternative'). what do you think?

Comment (ratneshraval) [reply app/javascript/search/common/commonUtils.js]: What Keith said plus same can be applied to address function.

**Comment by ratneshraval** [comment]
@sunilb0575 I didn't see any new test cases, can you add?

File: app/javascript/search/search_list.js [hunk 2]
```diff
@@ -1,28 +1,28 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 import {urlPrefixHelper} from '../helpers/url_prefix_helper.js.erb'
-import {checkForNA, checkValueForNull, addressStringValueOrNa, phoneNumberOrNa} from './common/commonUtils'
+import {checkForNA, checkValueForNull, respectiveFullAddressOrNA, respectiveNumberOrNA} from './common/commonUtils'
 
 export default class SearchList extends React.Component {
   render () {
     const searchResult = this.props.searchResults
+    const primaryPhoneRelation = 'primary'
```
Comment (ratneshraval) [inline app/javascript/search/search_list.js]: These constants are now in 3 places, so they need to move to a common location.

**Comment by ratneshraval** [comment]
@sunilb0575 I'll approve and merge now so we can verify in integration but those constants need to be extracted out and you can use same story number in new PR.

**Pull request merged**